receiver/redisreceiver/redis_metric.go

```diff
@@ -15,67 +15,79 @@
 package redisreceiver
 
 import (
-	"fmt"
 	"strconv"
 
-	metricspb "github.com/census-instrumentation/opencensus-proto/gen-go/metrics/v1"
+	"go.opentelemetry.io/collector/consumer/pdata"
 )
 
 // An intermediate data type that allows us to define at startup which metrics to
 // convert (from the string-string map we get from redisSvc) and how to convert them.
 type redisMetric struct {
```

Comment:
In host metrics we used directly the pdata for this https://github.com/open-telemetry/opentelemetry-collector/blob/master/receiver/hostmetricsreceiver/internal/scraper/loadscraper/load_metadata.go, is is possible to do the same?

Reply:
Good idea, I like it. Mind if I do this in a followup PR?